src/coreclr/src/tools/crossgen2/ILCompiler.Reflection.ReadyToRun/ReadyToRunSection.cs

```diff
@@ -3,39 +3,17 @@
 // See the LICENSE file in the project root for more information.
 
 using System;
-using System.Collections.Generic;
-using System.IO;
-using System.Text;
+
+using Internal.Runtime;
 
 namespace ILCompiler.Reflection.ReadyToRun
 {
     public struct ReadyToRunSection
     {
-        /// <summary>
-        /// based on <a href="https://github.com/dotnet/coreclr/blob/master/src/inc/readytorun.h">src/inc/readytorun.h</a> ReadyToRunSectionType
-        /// </summary>
-        public enum SectionType
```

Comment:
I have always found it very useful when the enums matched so that I can easily find occurrences both in the runtime and all of our tools. I know that the upper case names with underscores are too long, too verbose and ugly. So I wonder if we should move the runtime to use the names in the ReadyToRunSectionType too (in some other PR).

Reply:
One thing I'm worried about is that part of this problem is just different naming conventions for C++ vs. C# code. If you're mostly concerned about the enums as printed under "Type: " in the R2RDump output, we could relatively easily postprocess the enum values by changing Pascal casing to uppercase + underscores and prepending the <code>READYTORUN_SECTION_</code> prefix, that should mostly match (or we could make it fully match) the runtime constants. For the broader question of actually renaming the runtime constants to "C# style", I'd welcome @jkotas' opinion on this.

Reply:
I was more concerned about different namings in the code bases. There is no reason for not having the same enum member names in C++ if we use `enum class`

Reply:
I would be fine with using `enum class` for the C++ definitions.

Reply:
OK, thanks Jan and Jan for your feedback. I'll prepare a separate PR for renaming the CoreCLR constants.